dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -275,38 +287,14 @@ echo -n "[CHE] Switching to \"${CHE_OPENSHIFT_PROJECT}\"..."
 oc project "${CHE_OPENSHIFT_PROJECT}" &> /dev/null
 echo "done!"
 
-# -------------------------------------------------------------
-# If command == cleanup then delete all openshift objects
-# -------------------------------------------------------------
-if [ "${COMMAND}" == "cleanup" ]; then
-  echo "[CHE] Deleting all OpenShift objects..."
-  oc delete all --all
-  echo "[CHE] Cleanup successfully started. Use \"oc get all\" to verify that all resources have been deleted."
-  exit 0
-# -------------------------------------------------------------
-# If command == rollupdate then update Che
-# -------------------------------------------------------------
-elif [ "${COMMAND}" == "rollupdate" ]; then
-  echo "[CHE] Rollout latest version of Che..."
-  oc rollout latest che
-  echo "[CHE] Rollaout successfully started"
-  exit 0
-# ----------------------------------------------------------------
-# At this point command should be "deploy" otherwise it's an error
-# ----------------------------------------------------------------
-elif [ "${COMMAND}" != "deploy" ]; then
-  echo "[CHE] **ERROR**: Command \"${COMMAND}\" is not a valid command. Aborting."
-  exit 1
-fi
-
 # -------------------------------------------------------------
 # Deploying secondary servers
 # for postgres and optionally Keycloak
 # -------------------------------------------------------------
 
 COMMAND_DIR=$(dirname "$0")
 
-if [ "${CHE_MULTIUSER}" == "true" ]; then
+if [[ "${CHE_MULTIUSER}" == "true" ]] && [[ "${COMMAND}" != *"rollupdate"* ]]; then
```

Comment:
Will be this statement true if the command is `cleanup`?

Reply:
good catch Sergio! Fixed! Thanks